sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -98,32 +109,79 @@ class FileCatalog(Base):
     """File catalog table"""
 
     __tablename__ = "file_catalog"
+    __table_args__ = UniqueConstraint(
```

Comment:
I looked up what this does and chatgpt said this
```
The UniqueConstraint in SQLAlchemy is used to define a constraint at the table level that ensures the uniqueness of data across multiple columns. When you apply a UniqueConstraint to a set of columns, SQLAlchmey and the underlying database ensure that no two rows of the table contain the same combination of values in those columns. This is particularly useful when you want to enforce uniqueness across a combination of columns, rather than just a single column.
```
That's great that it does that for us. Nice!

Reply:
What is the result of this if you try and add a duplicate? Does it throw an error? I like this constraint in the database, but it will be something we need to handle in indexer

Reply:
Good point. Yes, I believe SQLAlchemy will throw an `IntegrityError`. I will check the indexer code to make sure it is handled properly.